ts/Stock/StockToolsBindings.ts

```diff
@@ -35,6 +34,7 @@ import NavigationBindings from '../Extensions/Annotations/NavigationBindings.js'
 import { Palette } from '../Core/Color/Palettes.js';
 import Series from '../Core/Series/Series.js';
 import U from '../Core/Utilities.js';
+import TimeCycles from '../Extensions/Annotations/Types/TimeCycles';
```

Comment:
Stock Tools Bindings shouldn't import full modules

PS: Just realized StockTools imports ALL annotations. It seems to be caused by importing NavigationBindings, which imports Annotations.. I will create a separate ticket, but still, please import types only